gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -68,6 +68,21 @@ class BaselineIdea extends AbstractBaselinePlugin {
             moveProjectReferencesToEnd(ideaModuleModel)
         }
 
+        // Suggest and configure the "save actions" plugin if Palantir Java Format is turned on.
+        if (project == project.rootProject) {
+            IdeaModel ideaRootModel = project.extensions.findByType(IdeaModel)
```

Comment:
we already have the ideaRootModel above

Reply:
yeah but it's needlessly trapped inside `afterEvaluate`. Basically I think the code above is beyond gross (it reapplies the same steps to the root model from every single subproject) and so will not touch it, with an eye to refactor it in another PR.